shared/routes/Repl/index.html

```diff
@@ -106,9 +107,23 @@
 		display: none;
 	}
 
+	h2 {
+		margin: 1em 0 0 0;
+		padding: 0 8px;
```

Comment:
It's to match the padding on the iframe document's body. Not sure if `rem` is guaranteed to mean the same thing across iframe boundaries